winterbreeze.asciidoc

```diff
@@ -14,7 +14,7 @@ the following, all user-visible changes are listed:
     * +monitors..by-name.+ is *invalid*
 
  - the 'toggle' command does not toggle between zero and the last non-zero value
-   anymore but simply between 0 and 1. For the former behaivour, one needs to use
+   anymore but simply between 0 and 1. For the former behaviour, one needs to use
```

Comment:
This behavior is purposely different from master, right? Since it's an incompatible change anyway, wouldn't it make sense (in terms of orthogonality) to just *not* offer toggling for integer settings at all, but only for bools?

Reply:
Right, and in general it seems preferable to have an error message instead of just different behaviour.

Reply:
See #250